File: 프로그래머스/unrated/181846. 두 수의 합/두 수의 합.java
import java.math.BigInteger;

class Solution {
    public String solution(String a, String b) {
        BigInteger a1 = new BigInteger(a);
		BigInteger b1 = new BigInteger(b);
        // String으로 표기된 a, b를 BigInteger를 이용해 큰 정수로 변환한다
		
		BigInteger tmp = a1.add(b1);
        // tmp에 두 값을 합한다
        String answer = tmp.toString();
        // tmp를 다시 String으로 변환한다
        return answer;
    }
}

File: 프로그래머스/0/181846. 두 수의 합/두 수의 합.js
function solution(a, b) {
    // 문자열로 된 a와 b를 BigInt를 이용해 큰 정수로 변환
    let a1 = BigInt(a);
    let b1 = BigInt(b);
        
    // a1과 b1을 더하여 tmp에 저장
    let tmp = a1 + b1;
        
    // tmp를 다시 문자열로 변환하여 반환
    let answer = tmp.toString();
        
    return answer;
}